apix_documentation_swagger_v4.yaml

```diff
@@ -0,0 +1,1186 @@
+        codes are sometimes sent to an end user's mobile phone before they are
+        able to log in. Verification codes cannot be re-used.
+      allOf:
+        - $ref: '#/components/schemas/GenericData'
+      type: object
+      required:
+        - attributes
+      properties:
+        attributes:
+          type: object
+          allOf:
+            - $ref: '#/components/schemas/MFA_Challenge'
+            - type: object
+              required:
+                - prompt
+              properties:
+                prompt:
+                  type: string
+                  description: The instructions to show to the owner of the
+                    account
+                  example: Please provide the code sent to your phone.
+
+    MFA_Challenge_Image:
+      description: Structure of an MFA Challenge of type 'image'
+      allOf:
+        - $ref: '#/components/schemas/GenericData'
+      type: object
+      required:
+        - attributes
+      properties:
+        attributes:
+          type: object
+          allOf:
+            - $ref: '#/components/schemas/MFA_Challenge'
+            - type: object
+              required:
+                - prompt
+              properties:
+                prompt:
+                  type: object
+                  properties:
+                    type:
+                      type: string
+                      description: "The type of the image: url or uri"
+                    value:
+                      type: string
+                      description: The URL of the image for url or the content
+                        of the image for uri
+                      example: http://via.placeholder.com/350x150
+
+    MFA_Challenge_Image_Choices:
+      description: Structure of an MFA Challenge of type 'image_choice'
+      allOf:
+        - $ref: '#/components/schemas/GenericData'
+      type: object
+      required:
+        - attributes
+      properties:
+        attributes:
+          type: object
+          allOf:
+            - $ref: '#/components/schemas/MFA_Challenge'
+            - type: object
+              required:
+                - prompt
+              properties:
+                prompt:
+                  type: array
+                  description: An array of images
+                  items:
+                    type: object
+                    properties:
+                      id:
+                        type: integer
+                        description: The ID of the option (basically the index
+                          of the array)
+                      type:
+                        type: string
+                        description: "The type of the image: url or uri"
+                      value:
+                        type: string
+                        description: The URL of the image for url or the content
+                          of the image for uri
+                        example: http://via.placeholder.com/350x150
```

Comment:
Same comments for this challenge

Reply:
@jdsampayo I think this is not yet implemented in Plunderer::Faker::Client, right?